packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -234,17 +240,24 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
             ...unhandledProps,
           })}
         >
-          <Input readOnly onClick={showCalendarGrid} value={valueFormatter(selectedDate)} />
+          <Input
```

Comment:
Please could you update this part to also use `createShorthand`, like other parts are doing?

Reply:
Should I then also add slots for Input?

Reply:
Yeah, let's do that :)